pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -88,20 +110,21 @@ public InputStream getStream() throws IOException {
     if (stream != null) {
       return stream;
     }
-    StreamWrapperFinalizeAction finalizeAction = this.finalizeAction;
+    LazyCleaner.CleaningAction finalizeAction = this.finalizeAction;
     if (finalizeAction != null) {
-      return finalizeAction.getStream();
+      InputStream stream = ((StreamWrapperCleaningAction) (finalizeAction)).getStream();
+      if (stream != null) {
+        return stream;
+      }
     }
 
     return new java.io.ByteArrayInputStream(castNonNull(rawData), offset, length);
   }
 
   @Override
   public void close() throws IOException {
-    StreamWrapperFinalizeAction finalizeAction = this.finalizeAction;
-    if (finalizeAction != null) {
-      finalizeAction.close();
-      this.finalizeAction = null;
+    if (cleaner != null ) {
```

Comment:
Frankly speaking, the spaces before `)` looks odd

Reply:
fixed